src/Query.php

```diff
@@ -103,7 +103,7 @@ private function datetime_to_utc_string($datetime) {
         if($datetime instanceof \DateTime) {
             $datetime_copy = clone $datetime;
             $datetime_copy->setTimezone(new \DateTimeZone('UTC'));
-            return $datetime_copy->format('Y-m-d\TH:i:s');
+            return $datetime_copy->format('Y-m-d\TH:i:s\Z');
```

Comment:
Hehe, PHP is funny, about [`DateTime::ISO8601`](http://php.net/manual/en/class.datetime.php#datetime.constants.iso8601), the manual says

> Note: This format is not compatible with ISO-8601, but is left this way for backward compatibility reasons. Use DateTime::ATOM or DATE_ATOM for compatibility with ISO-8601 instead.

`DateTime::ATOM` though, uses `+00:00` instead of `Z`, but as far as I understands, that is in line with ISO 8601. 

So maybe we should use that? Thinking of the scenario where the `setTimezone(new \DateTimeZone('UTC'))` is removed for some reason, but the format string isn't updated.

...but our tests will catch that, so let's just use `Z` like you did here. "`DateTime::ATOM`" is also a bit cryptic.